admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -186,6 +187,10 @@ export default class CreateOrderPage {
     });
   }
 
+  initSummaryOrderStatus() {
+    $(createOrderMap.summaryPaymentModule).trigger('change');
```

Comment:
You use the trigger to launch your change listener, but isn't it cleaner to init it manually using the first select option?
With
```
this.summaryManager.setOrderState($(`${createOrderMap.summaryPaymentModule} option:selected`).data('order-state'))
```

Reply:
Fixed